vgen_process_simulink_model.m

```diff
@@ -94,7 +94,7 @@
 
 quartus_workflow_cmd = "python3 " + mp.codegen_path + "/autogen_quartus.py -j " + config_file ...
     + " -w " + working_dir + " -l " + second_cmd + " exit &";
-system(quartus_workflow_cmd);
+%system(quartus_workflow_cmd);
```

Comment:
Is this a temporary comment to get a current project working?  If not, we should just get rid of this

Reply:
Not even that, I just don't want to wait for the quartus workflow stuff to run when I am not working on that so I comment it out and am quite good at forgetting to uncomment it before committing. I will get this updated

Reply:
Ah!  Gotcha.  I'm guilty of that too...

Reply:
Yep, that checks out. It doesn't seem like the Quartus workflow has been running for me lately when I try to do the autogen.

Reply:
Uncommented